<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="description"
        content="Foreseeing Survival Through Fuzzy Intelligence: A Cognitively-Inspired Incremental Learning Based de novo Model for Breast Cancer Prognosis by Multi-Omics Data Fusion.">
  <meta name="keywords" content="Breast cancer prognosis, Survival prediction, Multimodal feature fusion, Data-efficient learning, Machine learning">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Foreseeing Survival Through Fuzzy Intelligence: A Cognitively-Inspired Incremental Learning Based de novo Model for Breast Cancer Prognosis by Multi-Omics Data Fusion.</title>

  <link href="https://fonts.googleapis.com/css?family=Google+Sans|Noto+Sans|Castoro"
        rel="stylesheet">

  <link rel="stylesheet" href="./static/css/bulma.min.css">
  <link rel="stylesheet" href="./static/css/bulma-carousel.min.css">
  <link rel="stylesheet" href="./static/css/bulma-slider.min.css">
  <link rel="stylesheet" href="./static/css/fontawesome.all.min.css">
  <link rel="stylesheet"
        href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <link rel="stylesheet" href="./static/css/index.css">
  <link rel="icon" href="./static/images/Picture2.png">

  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script defer src="./static/js/fontawesome.all.min.js"></script>
  <script src="./static/js/bulma-carousel.min.js"></script>
  <script src="./static/js/bulma-slider.min.js"></script>
  <script src="./static/js/index.js"></script>
</head>
<body>

<nav class="navbar" role="navigation" aria-label="main navigation">
  <div class="navbar-brand">
    <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false">
      <span aria-hidden="true"></span>
      <span aria-hidden="true"></span>
      <span aria-hidden="true"></span>
    </a>
  </div>
  <div class="navbar-menu">
    <div class="navbar-start" style="flex-grow: 1; justify-content: center;">
      <a class="navbar-item" href="https://aviralchharia.github.io">
      <span class="icon">
          <i class="fas fa-home"></i>
      </span>
      </a>

      <div class="navbar-item has-dropdown is-hoverable">
        <a class="navbar-link">
          More Research
        </a>
        <div class="navbar-dropdown">
          <a class="navbar-item" href="https://aviralchharia.github.io/cAPTured/">
            cAPTured
          </a>
	  <a class="navbar-item" href="https://aviralchharia.github.io/Deep_Precognitive_Diagnosis/">
            Deep-Precognitive Diagnosis
          </a>	  
	  <a class="navbar-item" href="https://neurodiag.github.io/Schizo-Net/">
            Schizo-Net
          </a>
        </div>
      </div>
    </div>

  </div>
</nav>


<section class="hero">
  <div class="hero-body">
    <div class="container is-max-desktop">
      <div class="columns is-centered">
        </div>
      <div class="columns is-centered">
        <div class="column has-text-centered">
          <h1 class="title is-1 publication-title">Foreseeing Survival through Fuzzy Intelligence: Cognitively-Inspired Incremental Learning model for Breast Cancer Prognosis by Multiomics Data Fusion</h1>
          <div class="is-size-5 publication-authors">
            <span class="author-block">
              <a href="https://aviralchharia.github.io/">Aviral Chharia</a><sup>1</sup></span>&nbsp;&nbsp;&nbsp;&nbsp;
            <span class="author-block">
              <a href="https://scholar.google.co.in/citations?user=gL9gR-4AAAAJ&hl=en">Neeraj Kumar</a><sup>2</sup></span>
            </div>

	 <div class="is-size-5 publication-authors">
            <span class="author-block"><sup>1</sup>Thapar Institute of Engineering and Technology, India</span>
            <br>
            <br>
            <span><b style="color:rgb(255, 100, 100);">MICCAI Predictive Intelligence in Medicine (MICCAI PRIME), 2021</b></span>
          </div>

          <div class="column has-text-centered">
            <div class="publication-links">
              <!-- PDF Link. -->
              <span class="link-block">
                <a href="https://link.springer.com/chapter/10.1007/978-3-030-87602-9_22"
                   class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                      <i class="fas fa-file-pdf"></i>
                  </span>
                  <span>Paper</span>
                </a>
              </span>
              <span class="link-block">
                <a href="978-3-030-87602-9_22.pdf"
                   class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                      <i class="ai ai-arxiv"></i>
                  </span>
                  <span>PDF</span>
                </a>
              </span>
              <!-- Video Link. -->
              <!-- <span class="link-block">
                <a href="https://www.youtube.com/"
                   class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                      <i class="fab fa-youtube"></i>
                  </span>
                  <span>Video</span>
                </a>
              </span> -->
              <!-- Code Link. -->
              <span class="link-block">
                <a href="https://github.com/aviralchharia/Breast-Cancer-Prognosis/"
                   class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                      <i class="fab fa-github"></i>
                  </span>
                  <span>Code</span>
                  </a>
              </span>
            </div>
            
          </div>
        </div>
      </div>
    </div>
  </div>
</section>


<section class="section">
  <div class="container is-max-desktop">
    <!-- Abstract. -->
    <div class="columns is-centered has-text-centered">
      <div class="column is-four-fifths">
        <h2 class="title is-3">Abstract</h2>
        <div class="content has-text-justified">
          <p>
	         High-precision breast cancer prognosis is crucial for early disease identification, avoiding hazardous 
            side-effects of unnecessary therapies, and decreasing mortality rates through personalized and tailored
            treatment regimens. However, designing a prognosis model continues to be challenging, given the intricate
            relationship between distinct genetic attributes, varied clinical results of drug therapies, the noisy 
            nature of gene expressions, and the high-class imbalance seen in multimodal cancer data. Furthermore, because 
            labeled omics data collection is costly and requires highly-trained experts, the data available is very limited.
            This makes the design of the conventional machine and deep learning models incredibly challenging as they require
            large quantities of data for learning the underlying intricate patterns and would otherwise overfit, decreasing
            model precision. Moreover, all present models suffer from a ‘closed world assumption.’ These models, once trained,
            cannot be updated in real-time (when more omics data is available in the future) without a complete re-training.
            The present study is the first to introduce the ‘Fuzzy’ way towards Breast cancer prognosis, framing the task
            as an incremental learning problem. The proposed approach allows the model to continually update its learned
            feature space on a non-stationary multimodal data stream emulating the human brain’s remarkable quality to learn
            over time. We demonstrate the model’s ability to learn complex relationships between different multimodal attributes,
            training on severely imbalanced and limited data by mapping it to a high-dimensional ‘fused’ feature space. 
            The proposed model surpasses state-of-the-art machine learning (ML) models significantly. These results suggest
            that prediction through ‘fuzzy intelligence’ is a promising approach towards breast cancer prognosis.
            <br>
            <br>
          </p>
        </div>
      </div>
    </div>
  <figure class="image" id="fig-method">
  <img src="static/images/model_architecture.png" class="image-spacing, center" style="width: 750px">
  <br>
  <div class="columns is-centered has-text-centered">
  <div class="column is-four-fifths">
  <h2 class="title is-size-6">Architecture of the proposed Breast cancer prognosis model based on Fuzzy Incremental Learning 
	  for classifying patients as long and short-term survivors on multimodal omics data (i.e., gene expressions, copy number alteration & clinical).</h2>
  </div>
  </div>
  </div>
  </figure>
</section>


<section class="section">
  <div class="container is-max-desktop">
    <!-- A -->
    <div class="columns is-centered has-text-centered">
      <div class="column is-four-fifths">
        <h2 class="title is-3">Summary</h2>
        <div class="content has-text-justified">
        </div>
          <div class="content has-text-justified">
		  
     <figure class="image" id="fig-method">
      <img src="static/images/paradigm.png" class="center" style="width: 750px">
        <h2 class="title is-size-6">Illustration of the (a) Conventional prognosis models with closed world assumption,
		(b) The required re-training for conventional leaned feature space update (c) Proposed real-time update 
		of learned fused feature space.</h2>
        </figure>
       <p>
      Aptamers – short sequences of RNA, DNA, or peptides, have emerged as powerful tools with
      unique binding properties for specific target molecules. They hold immense potential especially
      in diagnostics, therapeutics, and biosensing. However, traditional lab-based methods for
      discovering aptamers are labor-intensive, costly and often fail to yield high-affinity aptamers.
      The current benchmark is SELEX which takes up to 02 years for finding a single high affinity
      Aptamer. The goal of this project was not only to develop a faster alternative of SELEX to
      accelerate the discovery of Aptamers and cut down the required time but also outperform it in
      terms of precision.
         <br><br>
      We developed “cAPTured”, a fuzzy continual machine learning model for predicting
      aptamer-target protein interactions. cAPTured used four distinct feature encodings: k-mer and
      revck-mer based aptamer-sequence encoding, as well as AAC and PseAAC based target
      protein-sequence encoding. These encoding methods were used to extract essential
      information from aptamer and protein sequences. Subsequently, cAPTured fused and
      embedded these encoded features into a low-dimensional latent space, preserving statistically
      significant features. Here, the goal was to design the model to be agnostic to distribution shifts,
      allowing it to adapt and update its learned feature space in real-time based on a non-stationary
      interaction-pair data stream.
         <br><br>
      Our results found that cAPTured cuts the time required for aptamer discovery from the current
      benchmark of around 2-3 years (which is required by SELEX) to just a few minutes. cAPTured
      outperforms existing benchmarks by exhibiting a 04% increase in precision when tested on
      benchmark lab datasets. Notably due to its inherent feature engineering and fuzzy neural
      network-based design, cAPTured excels in handling limited data scenarios (with just 400, 300 or
      even 200 training samples) and maintaining relevance over time (05 pairs of distribution shifts),
      preventing the model from becoming outdated as new aptamer-target interaction pairs are
      discovered. This was additionally verified by mapping a t-SNE plot. These results underscore
      the potential of cAPTured as a valuable tool in the field of bioinformatics.
         <br><br>
      In conclusion, using cAPTured, bioinformatics engineers can massively reduce the experiment
      duration to find a high affinity aptamer by 700x times from the current benchmark in addition to
      increasing the precision by 04%. cAPTured’s ability to adapt to distribution shifts and its robust
      performance in limited data scenarios position it as a versatile and enduring tool.
      <br>
      <br>
      </p>          
      <figure class="image" id="fig-method">
	 <img src="static/images/parametric.png" class="center" style="width: 500px">
         <h2 class="title is-size-6">Effect of variation in hyperbox (H) expansion coefficient on (a) Number of hyperbox (H) formed during model training (b) Model training time (sec).</h2>
      </div>
      </figure>
      </div>
    </div>
  </div>
</section>
  
  

<section class="section">
  <div class="container is-max-desktop">
    <div class="columns is-centered">
    <!-- Animation.
    <div class="columns is-centered">
      <div class="column is-full-width">
        <h2 class="title is-3">Animation</h2>

        <h3 class="title is-4">Re-rendering the input video</h3>
        <div class="content has-text-justified">
          <p>
            Using <span class="dnerf">Nerfies</span>, you can re-render a video from a novel
            viewpoint such as a stabilized camera by playing back the training deformations.
          </p>
        </div>
        <div class="content has-text-centered">
          <video id="replay-video"
                 controls
                 muted
                 preload
                 playsinline
                 width="75%">
            <source src="./static/videos/replay.mp4"
                    type="video/mp4">
          </video>
        </div>
      
      </div>
    </div>
    Animation. -->


    <!-- Concurrent Work.
    <div class="columns is-centered">
      <div class="column is-full-width">
        <h2 class="title is-3">Related Links</h2>

        <div class="content has-text-justified">
          <p>
            There's a lot of excellent work that was introduced around the same time as ours.
          </p>
          <p>
            <a href="https://arxiv.org/abs/2104.09125">Progressive Encoding for Neural Optimization</a> introduces an idea similar to our windowed position encoding for coarse-to-fine optimization.
          </p>
          <p>
            <a href="https://www.albertpumarola.com/research/D-NeRF/index.html">D-NeRF</a> and <a href="https://gvv.mpi-inf.mpg.de/projects/nonrigid_nerf/">NR-NeRF</a>
            both use deformation fields to model non-rigid scenes.
          </p>
          <p>
            Some works model videos with a NeRF by directly modulating the density, such as <a href="https://video-nerf.github.io/">Video-NeRF</a>, <a href="https://www.cs.cornell.edu/~zl548/NSFF/">NSFF</a>, and <a href="https://neural-3d-video.github.io/">DyNeRF</a>
          </p>
          <p>
            There are probably many more by the time you are reading this. Check out <a href="https://dellaert.github.io/NeRF/">Frank Dellart's survey on recent NeRF papers</a>, and <a href="https://github.com/yenchenlin/awesome-NeRF">Yen-Chen Lin's curated list of NeRF papers</a>.
          </p>
        </div>
      </div>
    </div>
    Concurrent Work. -->

  </div>
</section>


<section class="section" id="BibTeX">
  <div class="container is-max-desktop content">
    <h2 class="title">BibTeX</h2>
    <pre><code>@inproceedings{chharia2021foreseeing,
  title={Foreseeing Survival through ‘Fuzzy Intelligence’: A cognitively-inspired incremental learning based de novo model for Breast Cancer Prognosis by multi-omics data fusion},
  author={Chharia, Aviral and Kumar, Neeraj},
  booktitle={Predictive Intelligence in Medicine: 4th International Workshop, PRIME 2021, Held in Conjunction with MICCAI 2021, Strasbourg, France, October 1, 2021, Proceedings 4},
  pages={231--242},
  year={2021},
  organization={Springer}
}</code></pre>
  </div>
</section>


<footer class="footer">
  <div class="container">
    <div class="content has-text-centered">
      <a class="icon-link"
         href="978-3-030-87602-9_22.pdf">
        <i class="fas fa-file-pdf"></i>
      </a>
      <a class="icon-link" href="https://github.com/aviralchharia/Breast-Cancer-Prognosis" class="external-link" disabled>
        <i class="fab fa-github"></i>
      </a>
    </div>
    <div class="columns is-centered">
      <div class="column is-8">
        <div class="content">
          <p>
            This website is licensed under a <a rel="license"
                                                href="http://creativecommons.org/licenses/by-sa/4.0/">Creative
            Commons Attribution-ShareAlike 4.0 International License</a>. The website template is from the <a href="https://github.com/nerfies/nerfies.github.io">Nerfies</a> project page.
          </p>
        </div>
      </div>
    </div>
  </div>
</footer>

</body>
</html>
